compiler/tests/fail/x86-64/unaligned_slice_copy.jazz

```diff
@@ -0,0 +1,10 @@
+// Cannot copy by chunks of 4 bytes starting at offset 3 bytes
```

Comment:
Why this should fail ? We can now have unaligned access.

Reply:
Currently, the `copy` operator only supports aligned accesses.